product-backend/scripts/create_db.sql

```diff
@@ -0,0 +1,9 @@
+DROP DATABASE IF EXISTS TeamA_{{USERNAME}};
+CREATE DATABASE TeamA_{{USERNAME}};
+use TeamA_{{USERNAME}};
+
+-- US001
+create table JobRoles (
+	jobRoleId INT primary key AUTO_INCREMENT,
```

Comment:
for sql columns please use snake_case https://en.wikipedia.org/wiki/Naming_convention_(programming)